### PR TITLE
[bitnami/consul] Add configurable service session affinity

### PR DESCRIPTION
**Pull request by rasmus**
**Description of the change**

Be able to control the service `sessionAffinity` setting.

**Benefits**

Be able to control the service `sessionAffinity` setting.

**Possible drawbacks**

None known

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/consul/values.yaml [hunk 1]
```diff
@@ -251,6 +251,7 @@ service:
   serverPort: 8300
   consulDnsPort: 8600
   uiPort: 80
+  sessionAffinity: "None"
```
Comment (miguelaeh) [inline bitnami/consul/values.yaml:254]: Please add this change to the `values-production.yaml` too

File: bitnami/consul/templates/consul-service.yaml [hunk 2]
```diff
@@ -27,4 +27,5 @@ spec:
       protocol: "UDP"
       port: {{.Values.service.consulDnsPort}}   
   clusterIP: None
+  sessionAffinity: {{.Values.service.sessionAffinity}}
```
Comment (miguelaeh) [inline bitnami/consul/templates/consul-service.yaml:30]:
```suggestion
  sessionAffinity: {{ .Values.service.sessionAffinity }}
```

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you very much for the PR!!!
Please take a look at my comments

**Comment by rasmus** [comment]
@miguelaeh I have updated the PR with the changes. I aligned the use of spaces to match your comment as there was a mixture of the the in the file

**Review by miguelaeh — APPROVED** [review]
Thank you very much for addressing the changes!!!
LGTM!!!

**Pull request merged**